views/js/controller/ItemResultPreviewer/itemResultPreviewer.js

```diff
@@ -1,33 +1,31 @@
 /**
  * @author Moyon Camille <camille@taotesting.com>
  */
-define([
-    'taoItems/previewer/factory'
-], function (previewerFactory) {
-    'use strict';
+define(["taoItems/previewer/factory"], function(previewerFactory) {
+  "use strict";
```

Comment:
Please revert to 4 spaces instead of 2.
More coding style info is here: https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/79037064/TAO+Frontend+Coding+guide

Reply:
Thank you. 
Now I'm using these rules as my default prettier config